Crashing in hand_track_demo

log:
```
python hand_track_demo.py
2025-06-21 11:25:11.353 python[61529:3172498] WARNING: AVCaptureDeviceTypeExternal is deprecated for Continuity Cameras. Please use AVCaptureDeviceTypeContinuityCamera and add NSCameraUseContinuityCameraDeviceType to your Info.plist.
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
I0000 00:00:1750497911.963337 3172498 gl_context.cc:369] GL version: 2.1 (2.1 Metal - 88), renderer: Apple M3 Max
INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
W0000 00:00:1750497911.969673 3172616 inference_feedback_manager.cc:114] Feedback manager requires a model with a single signature inference. Disabling support for feedback tensors.
W0000 00:00:1750497911.974129 3172619 inference_feedback_manager.cc:114] Feedback manager requires a model with a single signature inference. Disabling support for feedback tensors.
W0000 00:00:1750497913.232925 3172615 landmark_projection_calculator.cc:186] Using NORM_RECT without IMAGE_DIMENSIONS is only supported for the square ROI. Provide IMAGE_DIMENSIONS or use PROJECTION_MATRIX.
Traceback (most recent call last):
  File "/Users/thomwolf/Documents/Github/reachy_mini/examples/hand_track_demo.py", line 60, in <module>
    hands = hand_tracker.get_hands_positions(img)
  File "/Users/thomwolf/Documents/Github/reachy_mini/examples/hand_tracker.py", line 24, in get_hands_positions
    results = self.hands.process(cv2.cvtColor(img, cv2.COLOR_BGR2RGB))
cv2.error: OpenCV(4.11.0) /Users/xperience/GHA-Actions-OpenCV/_work/opencv-python/opencv-python/opencv/modules/imgproc/src/color.cpp:199: error: (-215:Assertion failed) !_src.empty() in function 'cvtColor'
```

--- a/examples/hand_track_demo.py
+++ b/examples/hand_track_demo.py
@@ -5,12 +5,6 @@ from reachy_mini import ReachyMini
 import numpy as np
 from scipy.spatial.transform import Rotation as R
 
-from noise import pnoise1
-
-
-def smooth_movement(t, speed=0.5, scale=0.8):
-    return pnoise1(t * speed) * scale
-
 
 def draw_debug(img, palm_center):
     h, w, _ = img.shape
@@ -47,19 +41,14 @@ cap = cv2.VideoCapture(4)
 hand_tracker = HandTracker()
 pose = np.eye(4)
 euler_rot = np.array([0.0, 0.0, 0.0])
-kp = 0.3
+kp = 0.2
 t0 = time.time()
 with ReachyMini() as reachy_mini:
     try:
         while True:
             t = time.time() - t0
-            left_antenna = smooth_movement(t)
-            right_antenna = smooth_movement(t + 200)
 
             success, img = cap.read()
-            if not success:
-                continue
-
             hands = hand_tracker.get_hands_positions(img)
             if hands is None:
                 continue
@@ -77,10 +66,11 @@ with ReachyMini() as reachy_mini:
                 pose[:3, 3][2] = (
                     error[1] * 0.04
                 )  # Adjust height based on vertical error
+                pose[:3, 3][1] = (
+                    error[0] * 0.02
+                )  # Adjust height based on vertical error
 
-                # antennas = [left_antenna, right_antenna]
-                antennas = np.array([0, 0])
-                reachy_mini.set_position(head=pose, antennas=antennas)
+                reachy_mini.set_position(head=pose)
             cv2.imshow("test_window", img)
 
             cv2.waitKey(1)

--- a/examples/head_track_demo.py
+++ b/examples/head_track_demo.py
@@ -5,12 +5,6 @@ from reachy_mini import ReachyMini
 import numpy as np
 from scipy.spatial.transform import Rotation as R
 
-from noise import pnoise1
-
-
-def smooth_movement(t, speed=0.5, scale=0.8):
-    return pnoise1(t * speed) * scale
-
 
 def draw_debug(img, eye_center, roll):
     _eye_center = (eye_center.copy() + 1) / 2  # [0, 1]
@@ -63,12 +57,8 @@ with ReachyMini() as reachy_mini:
     try:
         while True:
             t = time.time() - t0
-            left_antenna = smooth_movement(t)
-            right_antenna = smooth_movement(t + 200)
 
             success, img = cap.read()
-            if not success:
-                continue
 
             eye_center, roll = head_tracker.get_head_position(img)
             if eye_center is not None:
@@ -86,8 +76,7 @@ with ReachyMini() as reachy_mini:
                     error[1] * 0.04
                 )  # Adjust height based on vertical error
 
-                antennas = [left_antenna, right_antenna]
-                reachy_mini.set_position(head=pose, antennas=np.array(antennas))
+                reachy_mini.set_position(head=pose)
             cv2.imshow("test_window", img)
 
             cv2.waitKey(1)

